solutions/CSES Increasing Subsequence II.mdx

```diff
@@ -2,10 +2,16 @@
 id: cses-1748
 source: CSES
 title: Increasing Subsequence II
-author: Benjamin Qi, Óscar Garries
+author: Benjamin Qi, Óscar Garries, Timothy Gao
 ---
 
-Coordinate compress, then range sum.
+Let $dp(i)$ be the number of increasing subsequences ending on the right at $i$.
+Notice that $dp(i)$ can be calculated as the summation of all $dp(j)$ such that $j$ is in the range $[0,i]$ and nums[$j$] < nums[$i$].
```

Comment:
Why is `nums[` not in latex? Also, the original array is called $a$ I think

Reply:
Ok. I'm pretty new the latex so I'm not sure what to put in latex. Thanks for pointing that out.

Reply:
I think I will keep the array name as "nums" though because that makes it more clear what I'm referring to.